features/cart.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='stylesheet' href='https://breezecss.netlify.app/styles/breeze.css' >
```

Comment:
Suggestion - Add author and desc meta tag

Reply:
okay, will add that. Thank you!!